lib/cryptoexchange/exchanges/bitexbook/services/market.rb

```diff
@@ -0,0 +1,45 @@
+module Cryptoexchange::Exchanges
+  module Bitexbook
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super ticker_url
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Bitexbook::Market::API_URL}/symbols/statistic?last_day=true"
```

Comment:
the ticker returned is 24h? do they have minutely (or better frequency) data endpoint?

Reply:
according to their api doc that have minutely like this:
https://api.bitexbook.com/api/v2/symbols/statistic?grouping=1
maybe we can grab the latest one 
but volume needs to refer to 1 day?

Reply:
that is one way but that's 2 calls for 1 ticker. Wondering if they have better endpoint that we can use.